ibm_catalog.json

```diff
@@ -502,8 +544,9 @@
 					"install_type": "fullstack"
 				},
 				{
-					"label": "Standard (Deploy on Red Hat OpenShift)",
-					"name": "standard",
+					"label": "Standard with sample application (Deploy on Red Hat OpenShift)",
+					"name": "standard-fully-config",
+					"index": 2,
```

Comment:
If you are not updating the stack for this variation in this PR, please remove the changes here, as this will create a brand new variation